src/modules/referral/services/service.ts

```diff
@@ -242,7 +242,7 @@ export class ReferralService {
           .multipliedBy(feePct)
           .multipliedBy(d.multiplier)
           .multipliedBy(new BigNumber(10).pow(18))
-          .dividedBy(this.appConfig.values.acxUsdPrice)
+          .dividedBy(d.acxUsdPrice)
```

Comment:
No, because only the [deposits with this column not null](https://github.com/across-protocol/scraper-api/pull/173/files#diff-ffeb9268c58aa039146f3c129f010a71b2dea4e103bbda111e697352a785dc8bR31) are considered